rs/rosetta-api/icrc1/ledger/src/main.rs

```diff
@@ -124,28 +121,9 @@ fn init_state(init_args: InitArgs) {
     })
 }
 
-#[cfg(not(feature = "next-migration-version-memory-manager"))]
-#[pre_upgrade]
-fn pre_upgrade() {
-    #[cfg(feature = "canbench-rs")]
-    let _p = canbench_rs::bench_scope("pre_upgrade");
-
-    let start = ic_cdk::api::instruction_counter();
-    let mut stable_writer = StableWriter::default();
-    Access::with_ledger(|ledger| ciborium::ser::into_writer(ledger, &mut stable_writer))
-        .expect("failed to encode ledger state");
```

Comment:
Since we're now serializing the state to stable memory using the Memory Manager, do we have some test that verifies that after installing this wasm, downgrading to a pre-V1 version of the ledger fails?

Reply:
We had those tests but I removed them when mainnet ledgers were upgraded to V1: https://github.com/dfinity/ic/pull/1764/files. I wonder if we should reintroduce them, also because this test is used by the ICP ledger, which is not yet on V1...

Reply:
Ah yes, I see. I think to keep them, we'd need to add additional entries to the `mainnet-canisters.bzl` file (or somewhere else). I think it would be useful to have, but we could add that in a separate PR, WDYT?